recipes/iterativewgcna/LICENSE.txt

```diff
@@ -0,0 +1,13 @@
+BSD 3-clause license
+Copyright (c) 2015-2019, conda-forge
```

Comment:
This is the wrong licence. It should be the one of the upstream package: https://github.com/cstoeckert/iterativeWGCNA/blob/master/LICENSE

The `licence` and `licence_family` fields should then be `GPL-2.0-only` and `GPL` respectively.

Reply:
Apologies, now corrected.